spec/requests/api/v1/github_webhook/filter_spec.rb

```diff
@@ -25,17 +30,73 @@
     let(:params) do
       {
         action: 'merged',
+        pull_request: pull_request
+      }
+    end
+
+    it 'sends a slack notification to a given channel with the PR notification' do
```

Comment:
I think the `it` sentence is wrong but the code is ok, we don't want to send a notification when a PR is closed, this is because we don't want to spam `code-review` channel with useless posts.